Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Material/Material.h

```diff
@@ -165,9 +173,18 @@ namespace AZ
             bool TryApplyPropertyConnectionToShaderEnable(
                 const MaterialPropertyValue & value,
                 const MaterialPropertyOutputId & connection);
+            bool TryApplyPropertyConnectionToInternalProperty(
+                const MaterialPropertyValue & value,
+                const MaterialPropertyOutputId & connection);
```

Comment:
Note to reviewers: all of the above TryApply functions are new, not just the last one. I forgot to remove the declarations before committing an earlier PR, so they were just undefined before. The definitions in cpp are new in this PR.